.github/workflows/win.yml

```diff
@@ -284,14 +284,34 @@ jobs:
           .\crystal-cross.exe build src/compiler/crystal.cr -Di_know_what_im_doing -Dwithout_playground --link-flags=/STACK:10000000
           mv crystal.exe bin/
 
+      - name: Download latest shards release
```

Comment:
It's not necessarily "latest" now

Reply:
I think that fits. The intention is to download the latest release.